scripts/build-package.sh

```diff
@@ -9,6 +9,7 @@ echo "building package: ${1}"
 
 pushd ${ROOT_DIR}/packages/${1}
     npm ci
+    npm run format
```

Comment:
Thank you for this! It was a pain when the build-all-packages script would fail due to linting errors and I had to go to each package and run this :)